modules/nf-core/sambamba/depth/meta.yml

```
@@ -32,6 +32,16 @@ input:
         pattern: "*.{bai}"
         ontologies:
           - edam: "http://edamontology.org/format_2572"
+  - - meta2:
+        type: map
+        description: |
+          Groovy Map containing sample information
+    - bed:
+        type: file
+        description: bed file
+        pattern: "*.{bed}"
+        ontologies:
+          - edam: "http://edamontology.org/format_3586" # bed12
```

Should this be http://edamontology.org/format_3003?
```suggestion
          - edam: "http://edamontology.org/format_3003" # BED
```